docs/sources/metainfo/component.xml

```diff
@@ -301,6 +301,23 @@
                       </para>
                     </listitem>
                   </varlistentry>
+                  <varlistentry>
+                    <term>cockpit-manifest</term>
+                    <listitem>
+                      <para>
+                        The software can be launched from the menus of
+                        the Cockpit admin interface.  The value of the
```

Comment:
Could you link "Cockpit" to the project page, so everyone can quickly look up which project this is about?